<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfigOther</class>
 <widget class="QWidget" name="frmConfigOther">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetLeft" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gboxCom">
        <property name="title">
         <string>串口配置 - 立即应用</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="0" column="2">
          <widget class="QLabel" name="labBaudRateA">
           <property name="text">
            <string>波特率A</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cboxPortNameD"/>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="labBaudRateD">
           <property name="text">
            <string>波特率D</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="cboxBaudRateA">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labBaudRateB">
           <property name="text">
            <string>波特率B</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4" rowspan="2">
          <widget class="QPushButton" name="btnInfo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>系统信息</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cboxPortNameA">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="cboxBaudRateB"/>
         </item>
         <item row="2" column="3">
          <widget class="QComboBox" name="cboxBaudRateC"/>
         </item>
         <item row="3" column="3">
          <widget class="QComboBox" name="cboxBaudRateD"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cboxPortNameC"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="labBaudRateC">
           <property name="text">
            <string>波特率C</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labPortNameB">
           <property name="text">
            <string>通信串口B</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cboxPortNameB"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labPortNameC">
           <property name="text">
            <string>通信串口C</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labPortNameA">
           <property name="text">
            <string>通信串口A</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labPortNameD">
           <property name="text">
            <string>通信串口D</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" rowspan="2">
          <widget class="QPushButton" name="btnComImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>传输示意图</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxNet">
        <property name="title">
         <string>网络配置  - 重启应用</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtTcpServerHost">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>3</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="txtTcpServerPort">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labTcpServerPort">
           <property name="text">
            <string>TCP端口</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labUdpListenPort">
           <property name="text">
            <string>UDP监听</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="txtUdpListenPort"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labTcpListenPort">
           <property name="text">
            <string>TCP监听</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="txtTcpListenPort"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labTcpServerHost">
           <property name="text">
            <string>TCP地址</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>测试TCP</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>测试UDP</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>325</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetRight" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cboxPortNameA</tabstop>
  <tabstop>cboxBaudRateA</tabstop>
  <tabstop>cboxPortNameB</tabstop>
  <tabstop>cboxBaudRateB</tabstop>
  <tabstop>cboxPortNameC</tabstop>
  <tabstop>cboxBaudRateC</tabstop>
  <tabstop>cboxPortNameD</tabstop>
  <tabstop>cboxBaudRateD</tabstop>
  <tabstop>btnComImage</tabstop>
  <tabstop>btnInfo</tabstop>
  <tabstop>txtTcpServerHost</tabstop>
  <tabstop>txtTcpServerPort</tabstop>
  <tabstop>txtTcpListenPort</tabstop>
  <tabstop>txtUdpListenPort</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
